frontend/App.tsx

```diff
@@ -38,6 +38,7 @@ export function App(props: AppProps) {
         ))}
       </head>
       <body>
+        {/* @ts-ignore */}
```

Comment:
When I installed `@types/react-router-dom` a typescript error started happening that I didn't want to deal with right now